includes/Database_Upgrader.php

```diff
@@ -104,20 +104,7 @@ public function __construct( Injector $injector ) {
 	 * @return void
 	 */
 	public function register() {
```

Comment:
Note that tests need updating now and are failing.

Reply:
Tests passing.